packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.template.ts

```diff
@@ -0,0 +1,54 @@
+import { html, when, slotted } from "@microsoft/fast-element";
+import { BreadcrumbItem } from "./breadcrumb-item";
+import { Breadcrumb } from "../breadcrumb";
+import { endTemplate, startTemplate } from "../patterns/start-end";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#(BreadcrumbItem:class)} component.
+ * @public
+ */
+export const BreadcrumbItemTemplate = html<BreadcrumbItem>`
+    <div role="listitem" class="listitem" part="listitem">
+        ${startTemplate}
+        <slot
+            ${slotted({
+                property: "defaultSlottedNodes",
+                filter: value =>
+                    (value.nodeType === 3 && value.textContent!.trim().length !== 0) ||
+                    value.nodeType === 1 ||
+                    false,
+            })}
+        ></slot>
+        <slot name="control">
+            ${when(
+                x => x.defaultSlottedNodes.length === 0,
+                html<BreadcrumbItem>`
+                    <a
+                        class="control"
+                        part="control"
+                        href="${x => x.href}"
+                        aria-current="${x => (x.isCurrent ? "page" : void 0)}"
+                    >
+                        <span class="content" part="content">
+                            ${x => x.name}
+                        </span>
+                    </a>
+                `
+            )}
+        </slot>
```

Comment:
Do we need both of these? It seems like we could essentially just have a single default slot with default content inside it. When you pass other content, it'll replace the default content (control) with the slotted content.

I might be missing something, but I don't see why we couldn't drop lines 13-21 and remove the name of the slot on line 22.